sync.go

```diff
@@ -35,31 +58,22 @@ func (lw *LibWallet) AddSyncProgressListener(syncProgressListener SyncProgressLi
 }
 
 func (lw *LibWallet) AddEstimatedSyncProgressListener(syncProgressListener EstimatedSyncProgressListener, logEstimatedProgress bool) {
-	syncProgressEstimator := SetupSyncProgressEstimator(
-		lw.activeNet.Params.Name,
-		logEstimatedProgress,
-		lw.GetBestBlock,
-		lw.GetBestBlockTimeStamp,
-		syncProgressListener,
-	)
-
-	lw.AddSyncProgressListener(syncProgressEstimator)
+	lw.estimatedSyncProgressListeners = append(lw.estimatedSyncProgressListeners, syncProgressListener)
 }
 
 func (lw *LibWallet) SyncInactiveForPeriod(totalInactiveSeconds int64) {
-	for _, syncProgressListener := range lw.syncProgressListeners {
-		if syncProgressEstimator, ok := syncProgressListener.(*SyncProgressEstimator); ok {
-			syncProgressEstimator.DiscardPeriodsOfInactivity(totalInactiveSeconds)
-		}
+
+	lw.totalInactiveSeconds += totalInactiveSeconds
+	if lw.connectedPeers == 0 {
+		// assume it would take another 60 seconds to reconnect to peers
+		lw.totalInactiveSeconds += 60
 	}
 }
 
-func (lw *LibWallet) ResetSyncProgressListeners() {
-	for _, syncProgressListener := range lw.syncProgressListeners {
-		if syncProgressEstimator, ok := syncProgressListener.(*SyncProgressEstimator); ok {
-			syncProgressEstimator.Reset()
-		}
-	}
+func (lw *LibWallet) ResetSyncProgressListener() {
```

Comment:
- This can become `initSyncData`, with same code in lines 99-117 of `dcrlibwallet.go`
- Function should probably not be exported to ensure it is only called internally from `spvSync` and `rpcSync`